MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -829,20 +829,18 @@ protected BandwidthMetricData onLoad(long mediaStartTimeMs, long mediaEndTimeMs,
       segmentData.setRequestUrl(segmentUrl);
       switch (dataType) {
         case C.DATA_TYPE_MANIFEST:
+        case C.DATA_TYPE_MEDIA_INITIALIZATION:
```

Comment:
Create a separate case for `DATA_TYPE_MEDIA_INITIALIZATION` that can set the request type to either `video_init` or `audio_init`. No need to call `setRequestMediaDuration` for that case.

Reply:
for what it's worth, here's how we handle init fragments in DASH.js - https://github.com/muxinc/mux-embed/blob/master/lib/dashjs-monitor.js#L100-L147. But it's basically what Scott said; `video_init` or `audio_init`.